config/tests/guest/libvirt/upstream.cfg

```diff
@@ -246,5 +239,87 @@ variants:
                 kernel_args='root=/dev/vda2 rw console=tty0 console=ttyS0,115200 init=/sbin/init initcall_debug selinux=0'
                 no with_virtio_scsi
                 only virtio_blk
-only virtio_net
-only qcow2
+
+    - vm:
+        kernel_args='root=/dev/sda2 rw console=tty0 console=ttyS0,115200 init=/sbin/init initcall_debug selinux=0'
```

Comment:
Has this worked as expected?  You might want to use `hvc0` instead.

Reply:
@mopsfelder yep it works!